pygfx/objects/_more.py

```diff
@@ -4,21 +4,73 @@
 
 
 class Group(WorldObject):
-    """A Group is almost identical to a WorldObject. Its purpose is
-    to make working with groups of objects syntactically clearer.
+    """A group of objects.
+
+    A Group is useful when manipulating the scene graph as children can be
+    jointly moved/scaled/rotated. It typically has no visual properties.
+
+    Parameters
+    ----------
+    geometry : Geometry
+        The data defining the shape of the object.
+    material : Material
+        The data defining the appearence of the object.
+    visible : bool
+        Whether the object is visible.
+    render_order : int
+        The render order (when applicable for the renderer's blend mode).
+    render_mask : str
+        Determines the render passes that the object is rendered in. It's
+        recommended to let the renderer decide, using "auto".
+    position : Vector
+        The position of the object in the world. Default (0, 0, 0).
```

Comment:
Mmh. In this case, we might also want to have a stub constructor that hides the arguments we remove here. Otherwise, they will still show up in an IDE and be suggested for auto-completion which may be confusing to newcomers.